app/services/weetsService.js

```diff
@@ -1,15 +1,14 @@
 const request = require('../utils/request');
 
 const weetsService = {
```

Comment:
There is no need to have an object here. You can use `exports` to define exportable methods, or, if you want to keep using module.exports, you could just define it like:
```js
const getRandomNumber = () => Math.floor(Math.random() * (9999 - 0 + 1) + 0);
const getOneRandomPhrase = async () => {
    const { data } = await request(`/${getRandomNumber()}/math`);
    return data; 
}

module.exports = getOneRandomPhrase;
```